### ISSUE TITLE
The Texas Chain Saw Massacre (1433140)

### ISSUE THREAD

**Issue by NTMan** [ValveSoftware/Proton#7027]
# Compatibility Report
- Name of the game with compatibility issues: The Texas Chain Saw Massacre
- Steam AppID of the game: 1433140

## System Information
- GPU: AMD Radeon 7900XTX
- Video driver version: [Mesa 23.3-dev](https://gitlab.freedesktop.org/mesa/mesa/-/commit/d7daf78f62817f5d2299d9cfc46528757aaae492)
- Kernel version: 6.5.0-rc6
- Link to full system information report as [gist](https://gist.github.com/NTMan/33cca4906328793206218192820779ec)
- Proton version: experimental-bleeding-edge-8.0-52959-20230818-pf3d1cc-w1a4ab4-d143eb8-vee87a4

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1433140.log](https://github.com/ValveSoftware/Proton/files/12380525/steam-1433140.log)
[steam-1433140.log](https://github.com/ValveSoftware/Proton/files/12380652/steam-1433140.log)

## Symptoms 
Any match
![20230818195515_1](https://github.com/ValveSoftware/Proton/assets/200750/6b730139-e6dc-4379-aa15-09b830a6afca)

![20230818195758_1](https://github.com/ValveSoftware/Proton/assets/200750/66947890-1bcf-4728-8fc5-e0d703b6f3c5)

are immediately ended after start with message "Network Error - The supplied credentials are invalid."
![20230818194141_1](https://github.com/ValveSoftware/Proton/assets/200750/f5543b5a-57a0-4334-8484-72a3132da2f1)
It's because of anticheat?

## Reproduction
Always

**Comment by Laim** [comment]
You get the Supplied Credentials are Invalid error on W10 as well, it's either a bug with the game or something with EAC.

**Comment by wuilmerj24** [comment]
same problem on window 10 and debian 12.

**Comment by Zorrototo** [comment]
As of today the game and EAC seem to work.

```
[18:56:12:976] [Windows] [EAC Launcher] [Info] [Connection] Connect result: No error (0) Response Code: 304 Destination IP: 52.222.174.31
[18:56:12:976] [Windows] [EAC Launcher] [Info] No update needed.
[18:56:12:976] [Windows] [EAC Launcher] [Info] Current progress: 100 (1/1).
[18:56:13:019] [Windows] [EAC Launcher] [Info] Starting Wine module mapping, Wine version: 8.0.
[18:56:13:815] [Windows] [EAC Launcher] [Info] Launcher finished with: 301, 'Easy Anti-Cheat successfully loaded in-game'.
[18:56:13:815] [Windows] [EAC Launcher] [Info] Successfully initialized the Easy Anti-Cheat service, waiting for game window to become visible.
[18:56:13:815] [Windows] [EAC Launcher] [Info] Unlocking the launch lock, allowing next game to launch to take place.
[18:56:13:832] [Windows] [EAC Launcher] [Info] Downloaded 1 bytes in 413 ms (0.00 KB/s)

```

For non Qwerty people, put English (Qwerty) keyboard layout as first in your system and you will be able to use your normal keyboard (current game issue, no key rebinding possible yet).

**Comment by Zorrototo** [comment]
There is no Steam overlay for me, like for the Insurgency Sandstorm game. probably as for Insurgency, the overlay opens in the EAC loading screen and doesn't spawn for the game window.

//EDIT: that's exactly what happens. I can see the overlay in the EAC loading window before the game starts.

It starts "start_protected_game.exe" this is where the Steam overlay opens, then it probably starts the actual game BBQClient.exe but there is no overlay there.

**Comment by wuilmerj24** [comment]
Its works.  But Dead by daylight not work F

**Comment by Zorrototo** [comment]
DBD works. Anyway this is off topic don't bring DBD here for no reason.

**Comment by wuilmerj24** [comment]
@Zorrototo  It doesn't work for me. It kicks me out of the game when it loads.

**Comment by Zorrototo** [comment]
Why are you continuing to post about DBD here? Please post on the DBD issue https://github.com/ValveSoftware/Proton/issues/519 :heart: 

//EDIT: you already did.

**Comment by Coolroo** [comment]
Just started getting this issue again

**Comment by Zorrototo** [comment]
Would you mind giving at least some information? What issue?

**Comment by Coolroo** [comment]
> Would you mind giving at least some information? What issue?

"Provided credentials are invalid" issue when joining a lobby, and on pressing a on the start menu I get the message
```
Easy Anticheat Error

EAC client integrity Violation. Error code 4: EOS_ACCTV_IntegrityCatalogMissingMainExecutable: Could not locate game executable entry in the catalogue. (BBQClient-Win64-Shipping.exe)

**Comment by Zorrototo** [comment]
You can delete the Proton prefix for the game at `~/.local/share/Steam/steamapps/compatdata/1433140/` first, then you can try to verify files for  `Proton 8`, for the `Proton EasyAntiCheat Runtime`, make sure Proton 8 is configured to run the game. And verify file for `Texas Chain Saw Massacre`, and after that try to launch the game and see if situation improves.

I played 4 hours ago so it works on my side.

**Comment by Coolroo** [comment]
> You can delete the Proton prefix for the game at `~/.local/share/Steam/steamapps/compatdata/1433140/` first, then you can try to verify files for `Proton 8`, for the `Proton EasyAntiCheat Runtime`, make sure Proton 8 is configured to run the game. And verify file for `Texas Chain Saw Massacre`, and after that try to launch the game and see if situation improves.
> 
> I played 4 hours ago so it works on my side.

Tried that exact set of steps, still getting the issue. Seems a lot of people have begun getting this issue from the newest update from what they're saying on the steam discussions

**Comment by Zorrototo** [comment]
Check the EAC log file maybe? Multiple file in there `~/.local/share/Steam/steamapps/compatdata/1433140/pfx/drive_c/users/steamuser/Application Data/EasyAntiCheat/`

If you have the bandwidth uninstall the game, restart Steam, Install the game.

**Comment by ShawermaBox** [comment]
I encounter the same issues when pressing "E" in an attempt to access the Main Menu:

`Easy Anticheat Error

EAC client integrity Violation. Error code 4: EOS_ACCTV_IntegrityCatalogMissingMainExecutable: Could not locate game executable entry in the catalogue. (BBQClient-Win64-Shipping.exe)`

Additionally, when entering the lobby, I am kicked out after a few seconds with the following error message:

`Provided credentials are invalid`

does the game work for you ATM @Zorrototo ?

**Comment by Zorrototo** [comment]
This error is when you have an issue with EAC I assume. Game works perfectly since the update they did after the initial release. I will launch game now and confirm that at this minute it works.

//EDIT: yep, it works.

//EDIT2: try that, make sure Proton 8 is installed as well as the Proton EasyAntiCheat Runtime in Steam, make sure Proton 8 is the version used to run the game, close Steam, delete the folder `~/.local/share/Steam/steamapps/common/TCSM/EasyAntiCheat/`, start Steam, and verify the game files from its properties, so it downloads the missing EAC files. Start the game, see if you download the EAC update, and if it fixed the issue.

//EDIT 3: as the other guy before you, look into the files you find in the EAC folder in the prefix to know what the EAC client is doing and getting as messages/errors. Previous guy didn't say anything so it didn't help anyone.

**Comment by ShawermaBox** [comment]
Unfortunately, I have requested a refund for the game and will need to wait 12 hours to receive the money. Once I receive the refund, I will repurchase the game and follow the steps you provided. I will then inform you of the outcome.

**Comment by ShawermaBox** [comment]
In response to your //EDIT 3: I don't recall deleting the game when I requested the refund. I'll check for the log file once i get home and provide it here.

As a side note, I would recommend putting your edits in a new comment instead of editing a posted comment. This makes it easier for me to reply to you, and I will receive a notification if you reply.

**Comment by Zorrototo** [comment]
No, editing is the way to go, multi posting is a big nono, unless someone already replied. It is on you if you keep a page open too long before replying and not seeing the edits. I reply in new post if there are new posts after mine. From the logs you replied at same minute as my last edit.

**Comment by ShawermaBox** [comment]
Hey, I have some good news – it worked! Here's what I did:

1-  I deleted all game files from my SD Card and the internal drive, and TCSM Tech Test folder.

2-  I downloaded the game onto the internal drive.

3- Then, I followed your instructions by manually setting Proton 8 in the game's properties. After that, I closed Steam, went to the game's directory, deleted the EasyAntiCheat folder, verified the game file integrity, and finally, ran the game.

And it just worked! Thanks for your help @Zorrototo!

**Comment by Zorrototo** [comment]
If you reinstalled the game after uninstalling it, you didn't need to delete the EAC folder and verify the files :D

**Comment by ShawermaBox** [comment]
True, it might not have been necessary, but I did it just to be sure.

**Comment by ShawermaBox** [comment]
I'm encountering another problem where the Steam overlay doesn't appear. Interestingly, it seems to function while the Easy Anti-Cheat window is initializing, but once the game launches, I'm unable to open the overlay by pressing Shift + Tab.

I attempted to use GameScope with the following Game Lunch Options command:
`gamescope -f -W 1920 -H 1080 -w 1920 -h 1080 -- %command%`

Although the Easy Anti-Cheat window launches in full screen, the issue persists within the game window. Do you have any suggestions or solutions for this problem?

**Comment by Zorrototo** [comment]
https://github.com/ValveSoftware/Proton/issues/7027#issuecomment-1694624534

No. It is not solvable problem.
